unconference/LMIC-know-how.md

```diff
@@ -0,0 +1,200 @@
+
+
+### Mohamed (he/him)
+
+Mohamed (he/him) is in his mid twenties. He is from Mali where he lives with his parents and several siblings. He has sporadic ad-hoc income and is a freelancer. He is a native French speaker with basic English knowledge. He has completed some undergraduate courses at the university and has engineering background. He leads a citizen-driven grassroots organization that performs investigative journalism, reporting on governmental policies around ecological issues like environmental sustainability and climate change, and social issues like femicides. Mohamed’s goals are to find a way to have the grassroots organisation become sustainable, including paying wages to staff and supporting infrastructure like cloud computing and computers. He would also like to receive travel grants to support additional investigative journalism. Mohamed has never received any grant before (neither local or international). Mohamed doesn’t know where to find grants and lacks familiarity with the administrative process involved; moreover, he has no experience in designing large-scale research projects. Some of the questions Mohamed has are: What are the requirements of a donor? How do I report to them? How can I use the money? How do I pay people? 
+
+## Learning modules
+
+### Module 1: Scope of the curriculum - who is this resource for and how to use it
+
+* Why are we doing this (mission statement)
+* How these materials can be used (teaching material), licensing (CC-BY-NC), attribution
+* Learner personas
+* Overview of problems when applying for grants (challenges these materials will address)
+    * The grant does not fit the remit of the call for proposals
+    * Writing and grammar skills
+    * Persons/Collaborators do not have the skills
+    * Unrealistic timeline or budget
+    * Application does not show experience with grant writing
+* What this material will NOT teach
+    * Comprehensive overview of grant writing (there are other grant writing workshops, and this won’t cover discipline-specific considerations)
+    * Complete information about program/project management and leadership (community management, managing personnel), and planning for organizational sustainability
+
+*Module 1 learning objectives:* 
+At the end of this module, learners should be able to:
+* Understand the scope of this curriculum
+* Understand how to use and reuse these materials 
+* Recognize the target the audience of this content
+* Identify common issues while seeking funding that this curriculum will address
+
+
+### Module 2: Preparing to apply for grants
+
+* What is a grant and what are the types of grants
+    * Competitive requests for applications (RFAs)
+    * Unsolicited submissions and pitching ideas
+* Meeting and getting to know funders (building trust, learning about program priorities)
+* Building a professional brand
```

Comment:
social media can be really important for some funders who approach researchers directly - this can be a way they find out who to approach